### PR TITLE
change order logic in onCreate to avoid useless objects

### PR DESCRIPTION
**Pull request by jlstoupy98**

### CODE REVIEW

File: app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java [hunk 1]
```diff
@@ -63,12 +63,7 @@
     @VisibleForTesting(otherwise=VisibleForTesting.PRIVATE)
     public static boolean enableNav = true;
 
-    DrawerLayout drawerLayout;
```
Comment (RoxaneBurri) [inline app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java:66]: Yes we don't need this lines !

File: app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java [hunk 2]
```diff
@@ -90,6 +85,16 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_description);
 
+        fAuth = FirebaseAuth.getInstance();
+        if (fAuth.getCurrentUser() == null) {
```
Comment (RoxaneBurri) [inline app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java:89]: So you put the authentification first ?

**Review by RoxaneBurri — APPROVED** [review]

File: app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java (hunk 2 above)
Comment (jlstoupy98) [reply app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java:89]: Yes exactly, I did that in all classes in such a way that it's the first thing that is done when we create the activity. It a bit more logic and it's avoid other useless objects creation.

File: app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java (hunk 1 above)
Comment (Antho1426) [inline app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java:66]: Yes, in each of our activities, we don't need those DrawerLayout-related lines, thanks for removing them!

File: app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java (hunk 2 above)
Comment (Antho1426) [reply app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java:89]: In fact, this is 1. a good necessary security check not to be left out and 2. this is now done in a logical way! Furthermore, it is now implemented in a consistent way across all our activities! Nice refactor, thanks for this Jean-Luc!

**Review by Antho1426 — APPROVED** [review]

File: app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java (hunk 1 above)
Comment (jlstoupy98) [reply app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java:66]: Yeah, it's cleaner !

**Pull request merged**